### PR TITLE
[3.2] docs(textarea): Revamped TextArea Overview

### PR DESCRIPTION
**Pull request by xristianstefanov**

### CODE REVIEW

File: components/textarea/overview.md [hunk 1]
```diff
@@ -12,13 +12,15 @@ position: 0
 
 The Telerik <a href ="https://www.telerik.com/blazor-ui/textarea" target="_blank">Blazor TextArea component</a> is a highly customizable multi-line text input area. It provides features like auto resizing based on the user input, floating label functionality and [events]({%slug textarea-events%}) to respond to user actions.
```
Comment (dimodi) [inline components/textarea/overview.md:13]:
```suggestion
The Telerik <a href ="https://www.telerik.com/blazor-ui/textarea" target="_blank">Blazor TextArea component</a> is a highly customizable multi-line text input area. It provides features like auto resizing based on the user input and [events]({%slug textarea-events%}) to respond to user actions.
```

File: components/textarea/overview.md [hunk 2]
```diff
@@ -12,13 +12,15 @@ position: 0
...
 
-To use the Telerik TextArea in your Blazor application:
-* add the `<TelerikTextArea>` tag.
-* set its `Value` (one and two-way data binding applicable) parameter to a string property.
+1. Add the `<TelerikTextArea>` tag to a Razor file.
 
->caption Basic TextArea with two-way value binding
+2. Set the `Value` parameter to the desired `string`. It supports one-way and two-way binding.
+
+3. (optional) Set the `Label` and `AutoSize` properties to adjust the TextArea height based on the user input.
```
Comment (dimodi) [inline components/textarea/overview.md:21]: Label ??

File: components/textarea/overview.md [hunk 3]
```diff
@@ -35,78 +37,50 @@ To use the Telerik TextArea in your Blazor application:
...
+## Appearance
 
-* `DebounceDelay` - `int` - Specifies the time in milliseconds between the last typed symbol and the updating of the value. The default value is 150ms.
+The TextArea component provides settings to control its appearance. [Read more about the Blazor TextArea appearance settings]({%slug TextArea-appearance%}).
 
-* `Enabled` - `bool`, defaults to `true` - whether the TextArea is enabled.
+## Events
 
-* Events - see the [Events]({%slug textarea-events%}) article for more information.
+The Blazor TextArea fires blur and value change events for further customizing its behavior. [Read more about the Blazor TextArea events]({%slug textarea-events%}).
```
Comment (dimodi) [inline components/textarea/overview.md:46]:
```suggestion
The Blazor TextArea fires **blur** and value **change** events to respond to user actions. [Read more about the Blazor TextArea events]({%slug textarea-events%}).
```

File: components/textarea/overview.md [hunk 4]
```diff
@@ -35,78 +37,50 @@ To use the Telerik TextArea in your Blazor application:
...
+| `Name` | `string` | The `name` attribute of the HTML element. It is usually required so the `AutoComplete` will be honored by the browser. |
+| `PlaceHolder` | `string` | A `string` that maps to the `placeholder` attribute of the HTML element. If a `Label` is defined, it will be shown instead of the placeholder when the input is not focused. |
+| `TabIndex` | `Nullable<int>` | Maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key. |
+| `Title` | `string` | Maps to the `title` attribute of the HTML element. You can use it to add a [tooltip]({%slug tooltip-overview%}). |
+| `ValidateOn` | `ValidationEvent` | Configures the event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs). |
```
Comment (dimodi) [inline components/textarea/overview.md:72]: default value

**Review by dimodi — CHANGES_REQUESTED** [review]
Fix basic example and instructions. Provide ValidateOn default value.

File: components/textarea/overview.md (hunk 2 above)
Comment (xristianstefanov) [reply components/textarea/overview.md:21]: It's forgotten there..

**Review by dimodi — APPROVED** [review]

**Pull request merged**